features/02_articles/validating_article_slugline.feature

```diff
@@ -0,0 +1,49 @@
+@articles
+Feature: Validating if article slugline is created out of the package headline
+  when slugline metadata is is not present in request payload.
+  In order to publish an article and access it via slug
+  As a HTTP Client
+  I want to be able to check if the article slug was generated properly
+
+  Scenario: Submitting request payload in ninjs format and publishing a package
+    Given I am authenticated as "test.user"
+    When I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/api/{version}/content/push" with body:
+    """
+      {"language": "en", "body_html": "<p>some html body</p>", "versioncreated": "2016-09-23T13:57:28+0000", "firstcreated": "2016-09-23T09:11:28+0000", "description_text": "some abstract text", "place": [{"country": "Australia", "world_region": "Oceania", "state": "Australian Capital Territory", "qcode": "ACT", "name": "ACT", "group": "Australia"}], "version": "2", "byline": "ADmin", "keywords": [], "guid": "urn:newsml:localhost:2016-09-23T13:56:39.404843:56465de4-0d5c-495a-8e36-3b396def3cf0", "priority": 6, "subject": [{"name": "lawyer", "code": "02002001"}], "urgency": 3, "type": "text", "headline": "Abstract html test", "service": [{"name": "Australian General News", "code": "a"}], "description_html": "<p><b><u>some abstract text</u></b></p>", "located": "Sydney", "pubstatus": "usable"}
+    """
+    Then the response status code should be 201
+    And I am authenticated as "test.user"
+    And I add "Content-Type" header equal to "application/json"
+    Then I send a "POST" request to "/api/{version}/content/routes/" with body:
+     """
+      {
+        "route":{
+          "name":"article",
+          "type":"content"
+        }
+      }
+     """
+    Then the response status code should be 201
+    And I am authenticated as "test.user"
+    And I add "Content-Type" header equal to "application/json"
+    Then I send a "POST" request to "/api/{version}/packages/5/publish/" with body:
```

Comment:
how do You know that it's fifth? First 4 are loaded in fixtures?

Reply:
yes

Reply:
preferably, the database should be empty by default, and we should create all needed resources using contexts, like creating a new tenant (`Given there is a new tenant ...`) etc. This way we would have more control over the data that is inserted into the database in the given scenario. But if the current implementation is ok we can stick to it for now.